graphql-api/src/queries/variant-datasets/gnomad-v4-variant-queries.ts

```diff
@@ -252,9 +252,11 @@ const shapeVariantSummary = (subset: any, context: any) => {
 
     const exomeFilters = variant.exome.filters || []
     const genomeFilters = variant.genome.filters || []
+    const jointFilters = variant.joint.filter || []
 
     const hasExomeVariant = variant.exome.freq[subset].ac_raw
     const hasGenomeVariant = variant.genome.freq[subset].ac_raw
+    const hasJointVariant = variant.joint.freq[subset].ac_raw
```

Comment:
One thing I got caught on a couple times while working on the non-UKB dataset is code similar to this pattern. Non-UKB doesn't have genomes so `variant.genome.freq[subset]` would be `undefined` for non-UKB, and then you'd get a crash when trying to dereference `ac_raw` on it.

Is that going to be a problem with the joints?

Reply:
Yep it is, thanks for catching this in review before our two branches met in production and joint's lack of a non-ukb subset caused problems